### PR TITLE
Initial commit for notebook template

### PR DESCRIPTION
**Pull request by ljhwang**
Initial commit for notebook template. 

Notebook sets style and standards for modules.  

It adds one asset / icon to be used with **Important** boxes. Please feel free to suggest a different one or make prettier.

This changes the initial template from what is stated in the README.  Minor changes to format of the header. Major change is moving the Table of Contents down such that the descriptive information is first.

I expect there are typos ;).  I expect this to be further refined with use :). 

@naliboff for review

### CODE REVIEW

File: bin/template/.ipynb_checkpoints/0_main_template-checkpoint.ipynb [hunk 1]
```diff
@@ -0,0 +1,111 @@
+{
```
Comment (naliboff) [inline bin/template/.ipynb_checkpoints/0_main_template-checkpoint.ipynb]: <p>Is <code>Boundary and Initial Conditions</code> the only planned subsection under <code>Model Setup</code>, or will more be added later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/3/discussion/#comment-1731894201'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (naliboff) [inline bin/template/.ipynb_checkpoints/0_main_template-checkpoint.ipynb]: <p>Maybe keep the van Zelst reference and then have something at the top like "The course materials are based on prior tutorials designed by Iris van Zelst, Mohammed Gouiza, etc following van Zelst et al. (2022)"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/3/discussion/#comment-1731894203'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

**Comment by naliboff** [comment]
@ljhwang - Thanks for adding these! I made a few comments via ReviewNB, but I forgot how tricky reviewing notebooks on github is.

General question - are some of the files (notebooks) on this PR currently duplicates?

**Comment by ljhwang** [comment]
@naliboff I think the problem is that the checkpoint got uploaded.  Let me fix this in the next pull request and see what happens.

**Comment by ljhwang** [comment]
@naliboff can you check that I squashed this correctly?  Then it is ready to merge.
ty

**Review by naliboff — APPROVED** [review]
👍

**Pull request merged**